apps/tailwind-components/pages/input/Ontology.story.vue

```diff
@@ -1,10 +1,15 @@
 <script setup lang="ts">
 import { ref } from "vue";
 
-const ontologySchemaId = ref<string>("pet store");
-const ontologyTableId = ref<string>("Tag");
-const modelValue = ref("green");
-const modelValueArray = ref(["colors", "mammals"]);
+// const ontologySchemaId = ref<string>("pet store");
+// const ontologyTableId = ref<string>("Tag");
+// const modelValue = ref("green");
+// const modelValueArray = ref(["colors", "mammals"]);
+
+const ontologySchemaId = ref<string>("CatalogueOntologies");
+const ontologyTableId = ref<string>("Countries");
+const modelValue = ref("Austria");
+const modelValueArray = ref(["Austria", "Brazil"]);
```

Comment:
I've changed this from the pet store as I needed something with more terms. By changing the pet store, it caused the e2e tests to break.This will be removed when this PR is ready for review.